packages/server/api/src/app/ai/chat/ai-mcp-chat.controller.ts

```diff
@@ -104,8 +104,18 @@ export const aiMCPChatController: FastifyPluginAsyncTypebox = async (app) => {
     });
 
     const tools = await getMCPTools();
-    const isAnalyticsLoaded = Object.keys(tools).includes('superset');
-    const isTablesLoaded = Object.keys(tools).includes('table');
+    logger.debug(
```

Comment:
We can remove the debug before merging, since this is called every time it's very spammy

Reply:
Same for the other debug logs here

Reply:
done